package.json

```diff
@@ -27,5 +27,8 @@
     "msw": "^2.7.0",
     "prettier": "^3.4.2"
   },
-  "packageManager": "yarn@4.5.3"
+  "packageManager": "yarn@4.5.3",
+  "dependencies": {
+    "expo-linear-gradient": "~14.0.2"
```

Comment:
this must be some kind of a mistake. It should be in apps/mobile

Reply:
true